sdk/src/RealUserMonitor.ts

```diff
@@ -0,0 +1,130 @@
+import {RealUserMonitoringEvents, Session, User} from "./Types";
+import {NativeEventEmitter, NativeModules, Platform} from 'react-native';
+import {setupNetworkMonitoring} from "./NetworkMonitor";
+import {getDeviceBasedId, log, warn} from "./Utils";
+
+const {RaygunNativeBridge} = NativeModules;
+const {osVersion, platform} = RaygunNativeBridge;
+
+const defaultURLIgnoreList = ['api.raygun.com', 'localhost:8081/symbolicate'];
+const SessionRotateThreshold = 30 * 60 * 100;
```

Comment:
Best to put a comment on what this value evaluates to in time (seconds, minutes etc)